wweval/R/ms_fig2.R

```diff
@@ -161,6 +161,14 @@ make_fig2_ct <- function(ww_quantiles,
       aes(x = date, y = log(calib_data)),
       color = "black", show.legend = FALSE
     ) +
+    geom_point(
+      data = quantiles_wide |> filter(.data$below_LOD == 1),
+      aes(x = date, y = log(calib_data)), color = "red", size = 1
+    ) +
+    geom_point(
+      data = quantiles_wide |> filter(.data$flag_as_ww_outlier == 1),
+      aes(x = date, y = log(calib_data)), color = "blue", size = 1
+    ) +
```

Comment:
Use shape as well as color for accessibility?

Reply:
Also, probably more ggplot-ish to handle these cases (for shape and/or color) with an aesthetic mapping rather than manual subsetting.

You could have a column called `observation_status` with values `standard`, `outlier`, `below_lod` and then map `color` and `shape` to `observation_status`.

I realize that here you have fill and/or color mapped to `model` elsewhere, but

A. Since there's no overlap between the discrete values of `model` and the discrete values of this `observation_status` column, you could have a shared manual scale for both.
B. In this particular case, `model` necessarily `== 'ww'`, so it's arguably more readable and ggplotish to do:
```R
model_color <- colors$model_colors$ww
``` 
and then pass that as a parameter (rather than as a mapped aesthetic) to geoms where you want the color/fill to reflect the wastewater model color choice, e.g.
```R
    geom_ribbon(
      aes(
        x = date, ymin = `0.025`, ymax = `0.975`,
      ),
      alpha = 0.2,
      show.legend = FALSE,
      fill = model_color
    ) 
```
C. If you ever do run into a case where you need two different scales for the same aesthetic, you can use [`ggnewscale`](https://eliocamp.github.io/ggnewscale/)

Reply:
Made those suggested adjustments, I think I'd rather keep the shapes as circles for now if thats ok... if we get comments in review process happy to change